client/src/components/Today/timeline/active-timeline-row.state.ts

```diff
@@ -0,0 +1,6 @@
+import { atom } from "jotai";
+
+/**
+ * @note this is an atom so that it's easier to reset from e.g.
```

Comment:
Make sure to reset it on navigation, even if just for completeness (it will basically always be reset due to the fact we set it to `null` on click outside, but that behavior may diverge eventually.